optimum/intel/ipex/modeling_base.py

```diff
@@ -128,10 +129,37 @@ def __init__(
         **kwargs,
     ):
         OptimizedModel.__init__(self, model=model, config=config)
-        # To do: add XPU support
-        self._device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
-        self._dtype = self.config.torch_dtype if self.config.torch_dtype is not None else torch.float32
+        device_map = kwargs.pop("device_map", None)
```

Comment:
`device_map` is used by `accelerate` to dispatch the weights for each layer across different devices which is not the case here so I think we should remove 

```suggestion
```